core/flickr_api_integration.py

```diff
@@ -40,8 +40,7 @@ def get_photo_files(photo_list):
     try:
         for photo in photo_list:
             request = requests.get(
-                f'https://live.staticflickr.com/{photo["server"]}/'
-                f'{photo["id"]}_{photo["secret"]}_b.jpg'
+                f'https://live.staticflickr.com/{photo["server"]}/' f'{photo["id"]}_{photo["secret"]}_b.jpg'
```

Comment:
```suggestion
                f'https://live.staticflickr.com/{photo["server"]}/{photo["id"]}_{photo["secret"]}_b.jpg'
```
If this is on one line, might as well make it 1 string.

Reply:
Oh this was what I'd had to do to put `# noqa` on things because the flake8 action was failing.